src/global/signal_handler.h

```diff
@@ -28,7 +29,7 @@ typedef void (*signal_handler_t)(int);
 # define sig_str(signum) sys_siglist[signum]
 #endif
 
-void install_sighandler(int signum, signal_handler_t handler, int flags);
+void install_sighandler(int signum, signal_action_handler_t handler, int flags);
```

Comment:
this change breaks the win32 build.
```
../src/global/signal_handler.h:22:46: error: 'siginfo_t' has not been declared
   22 | typedef void (*signal_action_handler_t)(int, siginfo_t*, void*);
      |                                              ^~~~~~~~~
```

see https://jenkins.ceph.com/job/ceph-dev-new-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=windows,DIST=windows,MACHINE_SIZE=gigantic/54417//consoleFull

Reply:
Oops, will add a guard. Thanks for pointing this out, Kefu!